test/parallel/test-fs-readfile-fd.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
-const common = require('../common');
+require('../common');
+const { fixturesDir } = require('../common/fixtures');
 const assert = require('assert');
 const path = require('path');
 const fs = require('fs');
-const fn = path.join(common.fixturesDir, 'empty.txt');
+const fn = path.join(fixturesDir, 'empty.txt');
```

Comment:
Hi @tobbik! Welcome, and thanks for this! Can you change this to use `fixtures.path()` instead of `fixtures.fixturesDir`? See the documentation at https://github.com/nodejs/node/tree/master/test/common#fixtures-module.

Reply:
Hi,

I created a new pull request with the changes as asked for.
https://github.com/nodejs/node/pull/16109

Reply:
Thanks, @tobbik. In the future, if you need to change a PR, you can make the changes on the same branch that you opened the PR with and push it up to GitHub. It's much easier/better for the review process. No worries, right now, though. I'll close this and move over to #16109.